tests/datacatalog/plugins/test_postgres.py

```diff
@@ -170,13 +170,15 @@ def test_storage_extract(event_loop, corpus, app):
     empty = event_loop.run_until_complete(retrieve_nothing())
     assert len(empty) == 0
 
+def get_id_sort_field(doc:dict)->str:
+    return doc['@id'] if '@id' in doc else ''
```

Comment:
Misschien mooier: `return doc.get('@id', '')`

Deze functie moet private zijn (beginnen met underscore) of misschien helemaal vervangen worden door een lokale lambda functie in `test_search_search()`